packages/SwingSet/test/test-transcript.js

```diff
@@ -12,11 +12,11 @@ async function buildTrace(c, storage) {
   // XXX TODO also copy transcript
```

Comment:
The new `getAllState` includes the full transcript, right? (In fact the old `getAllState` probably did too, just embedded in the kvStore rather than as a separate piece). If so, would it be safe to remove this TODO comment?

Reply:
Yup.  Will do.